Gemfile

```diff
@@ -6,7 +6,7 @@ ruby '2.3.1'
 gem 'foreman'
 
 # web server
-gem 'unicorn'
+gem 'puma'
```

Comment:
This was bundled with a version I tested/benchmarked pretty aggressively for concurrency issues so... if you don't mind, I'll make bumping gems a separate PR ?